libsql-server/src/lib.rs

```diff
@@ -101,6 +101,7 @@ pub struct Server<C = HttpConnector, A = AddrIncoming, D = HttpsConnector<HttpCo
     pub heartbeat_config: Option<HeartbeatConfig>,
     pub disable_namespaces: bool,
     pub shutdown: Arc<Notify>,
+    pub snapshot_at_shutdown: bool,
```

Comment:
can you move that to the `DbConfig` struct?

Reply:
Yup, done